Dockerfile

```diff
@@ -1,37 +1,76 @@
-FROM memgraph/memgraph:latest AS memgraph-mage
+FROM debian:buster as base
 
-FROM memgraph-mage
 USER root
 
-RUN apt-get update && \
-    apt-get --yes install curl git cmake g++ clang python3-dev uuid-dev && \
-    cd /usr/local/bin && \
-    ln -s /usr/bin/python3 python && \
-    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+#essentials for production/dev
+RUN apt-get update && apt-get install -y \
+    build-essential `mage-memgraph` \
+    cmake           `mage-memgraph` \
+    curl            `mage-memgraph` \
+    g++             `mage-memgraph` \
+    libcurl4        `memgraph` \
+    libpython3.7    `memgraph` \
+    libssl1.1       `memgraph` \
+    openssl         `memgraph` \
+    python3         `mage-memgraph` \
+    python3-pip     `mage-memgraph` \
+    uuid-dev        `mage-memgraph` \
+    clang           `mage-memgraph` \
+    --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
-# Install Rust
-RUN curl https://sh.rustup.rs -sSf | sh -s -- -y
-ENV PATH="/root/.cargo/bin:${PATH}"
 
-# Set mage as working directory
-WORKDIR /mage
-COPY . /mage
-
-# Build all necessary modules
-RUN python3 build
-RUN cp -r /mage/dist/* /usr/lib/memgraph/query_modules/
-
-# It's required to install python3 because auth module scripts are going to be
-# written in python3.
-RUN python3 -m  pip install -r /mage/python/requirements.txt
+#Memgraph
+RUN curl https://download.memgraph.com/memgraph/v1.6.1/debian-10/memgraph_1.6.1-community-1_amd64.deb --output memgraph.deb \
+    && dpkg -i memgraph.deb \
+    && rm memgraph.deb
 
-USER memgraph
 ENV LD_LIBRARY_PATH /usr/lib/memgraph/query_modules
+
 # Memgraph listens for Bolt Protocol on this port by default.
 EXPOSE 7687
+
 # Snapshots and logging volumes
 VOLUME /var/log/memgraph
 VOLUME /var/lib/memgraph
+# Configuration volume
 VOLUME /etc/memgraph
```

Comment:
Seems these are not necessary

Reply:
Yes it does but it is how it is on [here](https://github.com/memgraph/memgraph/blob/master/release/docker/memgraph_community.dockerfile) so I thought it would be good idea to add it here to. It serves just to show where are volumes, but it is not necessary.

Reply:
Remove them since they won't be used anymore, although you got the point of them being informatioal

Reply:
Removed it.